doc-content/enterprise-only/migration/migration-tool-niogit-7.4-proc.adoc

```diff
@@ -0,0 +1,217 @@
+[id='migration-tool-niogit-7.4-proc_{context}']
+
+= Using the migration tool on a `.niogit` directory
+
+The `.niogit` directory of your {CENTRAL} distribution contains all {CENTRAL} data. You can use the {CENTRAL} migration tool on your `.niogit` directory to prepare project data and system configurations for migration to {PRODUCT} 7.5.
+
+
+.Prerequisites
+* {PRODUCT} 7.5 is installed.
+* {PRODUCT} 7.4 in installed and {CENTRAL} is not running on {PRODUCT} 7.4.
```

Comment:
7.4 and 7.5 on the same machine, in different directories?

Reply:
Also you have to have 7.4 to migrate, with 7.2/7.3 this won't work? If so, I suggest pointing it out explicitly.

Reply:
Also no version variables, this doc forever remains for 7.4 and 7.5?

Reply:
Yes. That is correct. This update is for the 7.4 patching doc hence the static values. And this section is specifically about upgrading from 7.4 to 7.5. I think it's clear in the rendered docs.